client/src/components/FormComponents/InputField/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export const InputField = React.memo(props => {
```

Comment:
I'm not sure how much of a performance boost we get by trying to memoize this.  It's in the nature of inputs to change, and this will get re-rendered every time the input field is changed.  I suppose we get a boost if someone makes changes, then reverts them, etc such that we get the same inputs again, but am not sure if that offsets the processing time needed to compare the inputs every time InputField is called.  This is really an academic question though, as either case should be fast enough for us.